tic_tac_toe/game.py

```diff
@@ -0,0 +1,13 @@
+from tic_tac_toe.board import Board
+from tic_tac_toe.views import CommandLineBoardPresenter
+from tic_tac_toe.user_input import UserInput
+
+
+def run():
+    board = Board()
+    view = CommandLineBoardPresenter
```

Comment:
Hm, I feel like you want to be creating an instance of this class, not passing the class itself.

Reply:
After looking at this more closely I'm genuinely surprised it works when I run it locally! It doesn't make sense to me but it works! 😅